src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -2,10 +2,10 @@
 
 Release History
 ===============
-
-2.1.1
-+++++
-* Minor fixes.
+2.2.0
+++++++
+* BREAKING CHANGE: `vm/vmss create`, `vm/vmss identity assign/remove` will output `user assigned identities`
+                   in dictionary format
```

Comment:
What did it output before?

Reply:
Before:
```bash
  "userAssignedIdentities": ["/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/e2e/providers/Microsoft.ManagedIdentity/userAssignedIdentities/e2e-id", "/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/e2e/providers/Microsoft.ManagedIdentity/userAssignedIdentities/e2e-id2"]
```

After:
```bash
  "userAssignedIdentities": {
    "/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/e2e/providers/Microsoft.ManagedIdentity/userAssignedIdentities/e2e-id": {
      "clientId": "59f15e9d-9aed-4121-b4d1-fe29fa13f1eb",
      "principalId": "24e5a32c-c88b-40f0-9868-11f846b8c280"
    },
    "/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/e2e/providers/Microsoft.ManagedIdentity/userAssignedIdentities/e2e-id2": {
      "clientId": "0e8a38fa-c48e-4ab0-bbe4-7249cfd1c6d6",
      "principalId": "275b5b36-cec1-4d22-a097-ce40bc91ab5e"
    }
```
I originally wrote extra code to hide the new fields of 'clientId' and  'principalId', but backed out as those are new and important information users do want to see